test/conformance/kuttl/mutate/clusterpolicy/standard/existing/mutate-existing-node-status/01-assert.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kyverno.io/v1
```

Comment:
A Readme.md (test/conformance/kuttl/aaa_template_resources/README.md) could be helpful in understanding the tests.

Reply:
Done, added.